client/client.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+VARIANT_ENDPOINT = "http://rest.ensembl.org/vep/human/hgvs/{variant}" 
```

Comment:
This would normally be a ENV variable, but I figured that that detail wouldn't be so important.